### ISSUE TITLE
Import file warning

### ISSUE THREAD

**Issue by iFrostizz** [intellij-solidity/intellij-solidity#271]
Hey, first of all thanks for this plugin. It's definitely making my life easier.
After starting a contract inheriting from "Ownable.sol" from the OpenZeppelin library, I get an "import file" suggestion.
![image](https://user-images.githubusercontent.com/51274081/139055082-7716b952-6bb1-4c80-bbde-56886b237947.png)
It works great, but the warning doesn't stop after that the contract has been imported, and I think that it should.
Have a nice day

**Comment by SerCeMan** [comment]
Hey, @iFrostizz! The issues should be fixed in #280. I don't have a a Windows machine readily available, so it hasn't been tested. If you could try testing by cloning the latest HEAD and running a dev build, it would be greatly appreciated.

**Comment by iFrostizz** [comment]
Hi @SerCeMan, I've downloaded IntelliJ IDEA and cloned the repo. Then I built the plugin and opened it in an IDE for debugging purposes.
I've created a test file, with some imports. (I added the Ownable import manually)

![image](https://user-images.githubusercontent.com/51274081/146649263-753d28f7-698b-4586-b347-775bb1df09ae.png)

I'm not sure if I misconfigured anything, but the import file hint doesn't add an import path as intended, but shows an inspection instead.

![image](https://user-images.githubusercontent.com/51274081/146649507-096ef3a3-66ad-403e-b8cf-31f066679e54.png)

Thanks!
